compiler/hash-parser/src/parser/block.rs

```diff
@@ -120,13 +120,14 @@ impl<'stream, 'resolver> AstGen<'stream, 'resolver> {
         let subject = self.parse_expr_with_precedence(0)?;
 
         let mut cases = AstNodes::empty();
-
         let mut gen = self.parse_delim_tree(Delimiter::Brace, None)?;
 
+        // @@Future: re-work syntax to avoid always specifying a semi-colon!!
```

Comment:
Does this result in a change in syntax?

Reply:
Yes, it would, since it is expected that a `match` case is terminated with a `;`, but it is not necessarily needed.